main/utils.zkasm

```diff
@@ -928,6 +928,7 @@ handleError:
 handleBatchError:
     ; restore init state root and finish batch
     $ => SR         :MLOAD(batchSR)
+    $               :MLOAD(isLoadingRLP),JMPNZ(invalidTxRLP)
```

Comment:
Would be good to have a comment why RLP error processing is different

Reply:
This flag is not used outside the `zkevm-rom` so it could not cause any issue.
I would say the opposite, it could add information if the limit counters has been reached during the RLP